src/main/java/com/exasol/smalljsonfilesfixture/TestSetupDescription.java

```diff
@@ -0,0 +1,48 @@
+package com.exasol.smalljsonfilesfixture;
+
+import jakarta.json.bind.Jsonb;
+import jakarta.json.bind.JsonbBuilder;
+import lombok.*;
+
+/**
+ * This class describes a test setup.
+ */
+@Data
```

Comment:
To verify that the `equals()` method works (and to avoid that someone removes this), please add a unit test using [equalsverifier](https://jqno.nl/equalsverifier/)